source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -616,7 +631,7 @@ function Get-SettingsFromMof
 
     $mofContent = Get-MofContent -ReferenceConfiguration $ReferenceConfiguration
 
-    $mofContentFound = $mofContent.Where({$PSItem.ResourceID -match $Id})
+    $mofContentFound = $mofContent.Where( {$PSItem.ResourceID -match $Id})
```

Comment:
adding the space here isn't needed, should remove the space between the "(" and the "{".

Reply:
Removed whitespace